src/main/java/com/elytradev/concrete/recipe/RFIngredient.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2017:
+ * 	William Thompson (unascribed),
+ * 	Isaac Ellingson (Falkreon),
+ * 	Jamie Mansfield (jamierocks),
+ * 	and contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+ * of the Software, and to permit persons to whom the Software is furnished to do
+ * so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.elytradev.concrete.recipe;
+
+/**
+ * Reference IIngredient for RF. Any other kind of scalar ingredient will be nearly a carbon-copy of this one.
+ */
+public class RFIngredient implements IIngredient<Integer> {
```

Comment:
Why RF? Forge Energy/Units would make more sense imo

Reply:
I guess because of the Balancing standards of "RF is RF, Tesla is RF, Forge Energy is RF"? I do think EnergyIngredient would be better though to be system-agnostic.

Reply:
As @Boundarybreaker said, CSBS Interface Compatibility - some sort of energy cost, agnostic to actual implementation. RFIngredient is fine IMO, because it fits the text of that standard. Possibly a comment specifying the nature of this rule/generic nature of this class?

Reply:
So, rename to FUIngredient?

Reply:
I'd say `EnergyIngredient`, like @Boundarybreaker - it doesn't need to be tied to any specific energy system.

Reply:
EnergyIngredient would be fine.